packages/rocketchat-statistics/client/lib/rocketchat.js

```diff
@@ -0,0 +1,7 @@
+RocketChat.statistics = {};
+RocketChat.statistics.loadTimestamp = new Date();
+
+Meteor.call('getStatistics', true, function(error, statistics) {
```

Comment:
since this call is a bit heavy on server (with `true` as argument) and you need only the `uptime` property, I'll ask you to remove this call from here.. you can either do this call at `onCreated` hook of template `packages/rocketchat-ui-admin/client/admin.js` (you can remove the argument as well), or change it to get only the `uptime` from server.